vertexai/app/build.gradle.kts

```
@@ -65,7 +65,7 @@ dependencies {
     implementation("io.coil-kt:coil-compose:2.7.0")
     
     implementation("com.google.firebase:firebase-analytics:22.1.2")
-    implementation("com.google.firebase:firebase-vertexai:16.0.0-beta06")
+    implementation("com.google.firebase:firebase-vertexai:16.0.0")
```

The version will be updated by #1673 

```suggestion
    implementation("com.google.firebase:firebase-vertexai:16.0.0-beta06")
```